### PR TITLE
Tweaks for GCP backup/restore

### PR DESCRIPTION
**Pull request by chrisahl**
For issue https://issues.redhat.com/browse/AAP-13159

Would like https://github.com/ansible/aap-docs/pull/198 to be merged before this one.

### CODE REVIEW

**Review by itdove — APPROVED** [review]

**Review by leena-jawale — APPROVED** [review]

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc [hunk 1]
```diff
@@ -50,21 +31,23 @@ gcp_backups_delete:
     gcp_bucket_backup_name:
 ----
 
-With backup_names like ["backup1","backup2"]
+Parameter *backup_names* MUST specify an array of strings.  For example: ["backup1","backup2"].
```
Comment (ianf77) [inline stories/topics/proc-gcp-deleting-backups-playbook.adoc:34]: backuup_names -> do you mean gcp_backups_delete? I can't find a backup_names.

The backup-names  (in backticks) parameter must specify and array of strings, for example, ["backup1", "backup2"]

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc [hunk 2]
```diff
@@ -50,21 +31,23 @@ gcp_backups_delete:
...
+Parameter *delete* MUST specify *true* to verify the deletion should occur.  Otherwise the delete will be a dry run
```
Comment (ianf77) [inline stories/topics/proc-gcp-deleting-backups-playbook.adoc:35]: The delete (in backticks) parameter must be set to `true` to successfully delete.

Where does this parameter come from?

File: stories/topics/ref-gcp-populate-restore-file.adoc [hunk 3]
```diff
@@ -22,6 +22,7 @@ Provide values for the following parameters:
 * `gcp_existing_vpc` must be set to `false`.
 * `cloud_credentials_path` is the path for your Google Cloud service account credentials file.
 * `deployment_name` is the name under which the deployment must be restored. A new deployment will be created with this name. A deployment must not already exist with this name.
+* `backup_name` is the name of the backup in the bucket.  This name is displayed while using the `gcp_backup_deployment` or `gcp_backup_list` commands.
```
Comment (ianf77) [inline stories/topics/ref-gcp-populate-restore-file.adoc:25]: shouldn't it be gcp_backup_name  ?

Comment (chrisahl) [reply stories/topics/ref-gcp-populate-restore-file.adoc:25]: No, the parameter is correct as is

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc (hunk 2 above)
Comment (chrisahl) [reply stories/topics/proc-gcp-deleting-backups-playbook.adoc]: Fixed `delete`

File: stories/topics/proc-gcp-deleting-backups-playbook.adoc (hunk 1 above)
Comment (chrisahl) [reply stories/topics/proc-gcp-deleting-backups-playbook.adoc]: Fixed `backup_names`

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by ianf77** [comment]
Conflicts to fix

**Comment by chrisahl** [comment]
@ianf77 I fixed the merge conflict.  Should be good to merge now

**Pull request merged**